openstack/nova/templates/db-migrate-job.yaml

```diff
@@ -16,23 +16,21 @@ spec:
     spec:
       restartPolicy: OnFailure
       {{- include "utils.proxysql.job_pod_settings" . | indent 6 }}
+      initContainers:
+      {{- $dependencies := dict }}
+      {{- if not .Values.cell2.enabled }}
+        {{- $_ := set $dependencies "services" (print .Release.Name "-mariadb," .Release.Name "-api-mariadb") }}
+      {{- else }}
+        {{- $_ := set $dependencies "services" (print .Release.Name "-mariadb," .Release.Name "-api-mariadb," .Values.cell2.name "-mariadb") }}
```

Comment:
With changing from `.Values.mariadb_cell2.name` to `.Values.cell2.name`, we need the `.Release.Name` prefix again.